daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/ContractStateMachine.scala

```diff
@@ -487,15 +512,14 @@ object ContractStateMachine {
     *    all operations involving a contract with a key update this map.
     */
   final case class ActiveLedgerState[+Nid](
+      locallyCreatedThisTimeline: Set[ContractId],
```

Comment:
please add a comment explaining what exactly this tracks and how it is different from locallyCreated.